test/Google_Api_Test/src/Main.enso

```diff
@@ -10,7 +10,7 @@ main filter=Nothing =
 
 
 add_specs suite_builder =
-    suite_builder.group "Google Spreadsheets" group_builder->
+    suite_builder.group "Google Spreadsheets" pending="outdated" group_builder->
```

Comment:
seems unrelated

Reply:
Related. NI would potentially crash on this test and we know it's broken.